moped-editor/src/views/staff/components/NonLoginUserActivationButtons.js

```diff
@@ -59,7 +60,8 @@ const NonLoginUserActivationButtons = ({
   };
 
   const handleUpdateNonLoginUser = async () => {
-    const { password, __typename, ...restOfFormValues } = formValues;
+    const { password, __typename, ...restOfFormValues } =
+      transformFormDataIntoDatabaseTypes(formValues);
```

Comment:
i uncovered a bug! this was causing non-login-user's role name to be saved as a string instead of an array when editing non-login users. this only recently surfaced because a different bug was blocking non-login-user editing. you can undo this change and test it locally.